form_urlencoded/src/lib.rs

```diff
@@ -116,16 +116,33 @@ impl<'a> Iterator for ParseIntoOwned<'a> {
 
 /// The [`application/x-www-form-urlencoded` byte serializer](
 /// https://url.spec.whatwg.org/#concept-urlencoded-byte-serializer).
+/// Converts spaces (b' ') to the percent-encoded equivalent ("%20").
 ///
 /// Return an iterator of `&str` slices.
 pub fn byte_serialize(input: &[u8]) -> ByteSerialize<'_> {
-    ByteSerialize { bytes: input }
+    ByteSerialize {
+        bytes: input,
+        space_as_plus: false,
+    }
+}
+
+/// The [`application/x-www-form-urlencoded` byte serializer](
+/// https://url.spec.whatwg.org/#concept-urlencoded-byte-serializer).
```

Comment:
This should link to https://url.spec.whatwg.org/#string-percent-encode-after-encoding

Reply:
correct, changed the link in the function above as well.